sdk/js/src/Client.ts

```diff
@@ -36,7 +37,7 @@ export class DVCClient implements Client {
     logger: DVCLogger
     config?: BucketedUserConfig
     user: DVCPopulatedUser
-    private store: Store
+    private store: Store | ReactNativeStore
```

Comment:
could we make an abstract class so we don't have to do this?

Reply:
yeah! good callout :)